helm/fluent-logshipping-app/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 1.9.8
+appVersion: 2.1.6
```

Comment:
Wow, that's a big gap! No compatibility issues?

Reply:
That's what I need to test :)